pywinauto/findwindows.py

```diff
@@ -255,12 +257,14 @@ def _title_match(w):
         elements = [elem for elem in elements if elem.enabled]
 
     if best_match is not None:
+        # Build a list of wrapped controls.
+        # Speed up the loop by setting up local pointers
         wrapped_elems = []
+        add_to_wrp_elems = wrapped_elems.append
+        wrp_cls = backend_obj.generic_wrapper_class

```

Comment:
It depends on calls. Mostly it doesn't change too much but sometimes you see the difference.
This is an example of line profiling in ipython on my py27 x64:
```
%lprun -f pywinauto.findwindows.find_elements dlg.TabControl.select("Views")
```

```
   262         2            7      3.5      0.0          wrapped_elems = []
   263        65          328      5.0      0.0          for elem in elements:
   264        63          212      3.4      0.0              try:
   265        63        31850    505.6      4.1                  wrapped_elems.append(backend_obj.generic_wrapper_class(

```
```

   264         2            8      4.0      0.0          wrp_cls = backend_obj.generic_wrapper_class
   265        65          329      5.1      0.0          for elem in elements:
   266        63          215      3.4      0.0              try:
   267        63        30654    486.6      2.9                  add_to_wrp_elems(wrp_cls(elem))
```